SCHEMA-SPEC.md

```diff
@@ -24,15 +24,129 @@ None.
 
 #### Properties
 
-* `description` (optional): An informational description of the purpose of this schema.
-* `schema-url` (optional): A URL where someone may go to find more information about this schema. It is not meant for mechanical processing.
+None.
 
 #### Children
 
+* [`info`](#info-node) - one info node for that describes the schema itself.
 * [`node`](#node-node) - zero or more toplevel nodes for the KDL document this schema describes.
+* [`definitions`](#definitions-node) (optional): Definitions of nodes, values, props, and children block to reference in the toplevel nodes.
 * `node-names` (optional): [Validations](#validation-nodes) to apply to the _names_ of child nodes.
 * `other-nodes-allowed` (optional): Whether to allow nodes other than the ones explicitly listed here. Defaults to `false`.
 
+### `info` node
+
+The `info` node describes the schema itself.
+
+#### Values
+
+None.
+
+#### Properties
+
+None.
+
+#### Children
+
+* [`title`](#title-node): one or more titles
+* [`description`](#description-node) (optional): zero or more descriptions
+* [`author`](#author-and-contributor-nodes): one or more authors
+* [`contributor`](#author-and-contributor-nodes) (optional): zero or more contributors
+* [`link`](#link-node) (optional): zero or more URLs
+* [`license`](#license-node) (optional): zero or more licenses
+* [`published`](#published-and-modified-nodes) (optional): a publication date
+* [`modified`](#published-and-modified-nodes) (optional): a modification date
+* [`version`](#version-node) (optional): a (semver) version number
+
+### `title` node
+
+The title of the schema or the format it describes.
+
+#### Values
+
+* Title
+
+#### Properties
+
+* `lang` (optional): An IETF BCP 47 language tag
+
+### `description` node
+
+A description of the schema or the format it describes.
+
+#### Values
+
+* Description
```

Comment:
The definition for `info` says this can be "zero or more". I think I'd rather this be min1max1 instead of several blocks of text? Is there a use case for that?

Reply:
Multiple languages is what I had in mind. But there's no `max` validator that requires at most one for each value of a certain property.